docs/docs/guides/server-deployment.md

```diff
@@ -202,6 +202,22 @@ To store logs using GCP Logging, set the `DSTACK_SERVER_GCP_LOGGING_PROJECT` env
 
     </div>
 
+## Metrics
+
+If enabled, `dstack` collects and exports Prometheus metrics from running jobs. Metrics for jobs from all projects are available
+at the `/metrics` path, and metrics for jobs from a specific project are available at the `/metrics/project/<project-name>` path.
+
+By default, metrics are disabled. To enable, set the `DSTACK_ENABLE_PROMETHEUS_METRICS` variable.
+
+Each sample includes a set of `dstack_*` labels, e.g., `dstack_project_name="main"`, `dstack_run_name="vllm-llama32"`.
+
+Currently, `dstack` collects the following metrics:
+
+* A fixed subset of NVIDIA GPU metrics from [DCGM Exporter :material-arrow-top-right-thin:{ .external }](https://docs.nvidia.com/datacenter/dcgm/latest/gpu-telemetry/dcgm-exporter.html){:target="_blank"}.
+`dcgm-exporter` and `libdcgm` must be installed on the instance to enable these metrics.
+On AWS, Azure, GCP, and OCI backends the required packages are already installed.
+If you use SSH fleets, install `datacenter-gpu-manager-4-core` and `datacenter-gpu-manager-exporter`.
```

Comment:
What about other non-ssh VM backends like Lambda, Tensordock, Vultr, etc? Seems they don't support dcgm metrics, but the docs can be interpreted like users can install something to make it work.